shared/actions/platform-specific/index.desktop.tsx

```diff
@@ -234,29 +252,60 @@ const sendKBServiceCheck = (state: Container.TypedState, action: ConfigGen.Daemo
   }
 }
 
+/*
+ * Check if the client is out of date via the service -> api server
+ * There are three cases we care about from the frontend
+ * 1. OK
+ * 2. SUGGESTED
+ * 3. CRITICAL
+ */
+const startupCheckDelay = 2 * 60 * 1000 // 2 minutes
+const updateCheckInterval = 60 * 60 * 1000 // 1 hour
 function* startOutOfDateCheckLoop() {
-  while (1) {
-    try {
-      const toPut = yield checkForUpdate()
-      yield Saga.put(toPut)
-      yield Saga.delay(3600 * 1000) // 1 hr
-    } catch (err) {
-      logger.warn('error getting update info: ', err)
-      yield Saga.delay(3600 * 1000) // 1 hr
+  // don't bother checking during startup
+  yield Saga.delay(startupCheckDelay)
+
+  while (true) {
+    const action = yield checkForUpdate()
+    yield Saga.put(action)
+    // Only want to make a single check on mobile
```

Comment:
@chrisnojima Carried this over from the previous file, so have been making only a single check on mobile.

1. These timers should probably move to `actions/config/index.tsx` since we do want to check on mobile
2. We probably don't want mobile devices making a critically out of date check every hour to save on data usage (this is the reasoning I understood when reading the code originally).